.github/workflows/main.yml

```diff
@@ -18,16 +18,10 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements-dev.txt
-      - uses: isort/isort-action@v1.1.1
-        with:
-          sort-paths: antarest, tests
-          requirementsFiles: "requirements-dev.txt"
-      - name: Check with black
-        uses: psf/black@stable
-        with:
-          # Version of Black should match the versions set in `requirements-dev.txt`
-          version: "~=25.1.0"
-          options: --check --diff
+      - name: Check format (ruff)
+        run: |
+          ruff check antarest/ tests/
```

Comment:
Is it possible to define the directories in pyproject.toml ?
Would be more convenient to ensure consistency between CI and local execution.

Reply:
Actually it does not seem very convenient, you can only define patterns in the `include` configuration property, not simply directories.